@here/olp-sdk-dataservice-read/lib/DataStoreRequestBuilder.ts

```diff
@@ -46,6 +46,16 @@ async function addBearerToken(
 }
 
 export class DataStoreRequestBuilder extends RequestBuilder {
```

Comment:
Please add a comment about the class. It's an implementation of  RequestBuilder, uses to configure the instance with base-url to the service and implementation of downloadBlob and download methods

Reply:
Fixed